cv32e40x/sim/Common.mk

```diff
@@ -15,14 +15,24 @@ export SHELL = /bin/bash
 
 CV_CORE_REPO   ?= https://github.com/openhwgroup/cv32e40x
 CV_CORE_BRANCH ?= master
-CV_CORE_HASH   ?= 07c8644ed08e0caaa19848bc218632e9f2e75675
+CV_CORE_HASH   ?= ed7cfdfdcd8c86cd2df41dfad25810997221756e
 CV_CORE_TAG    ?= none
```

Comment:
Just a comment here - the sha_hash can be abbreviated to the most significant 7 hex digits (e.g. `ed7cffd`).  Makes life easier for the humans if you ever need to trace to a specific hash in github.

Reply:
Thanks.  Will try to remember.  It's too easy to copy-paste full hash from gitk!